### DISCUSSION TITLE
Tractor + FEATURE

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1310] [Art] [answered]
![Schermata 2022-08-07 alle 13 55 11](https://user-images.githubusercontent.com/58088988/183289330-03ef08b4-7994-488f-86c9-d5e9bd41c74d.png)

![Schermata 2022-08-07 alle 15 59 43](https://user-images.githubusercontent.com/58088988/183294434-d5213022-2da9-4bae-9809-c1facf656aa5.png)

I know it's not designed from the point of view of buildings, but it could be useful somewhere else

### **FEATURE**
**Our plants are harvested and re-seeded automatically**
_**When you click the tractor you will have to set which seed to use constantly, obviously the seeds will have to be bought previously**_

![image](https://user-images.githubusercontent.com/58088988/183289108-1e3021be-4775-4241-b043-74d0b0bfdce2.png)

![image](https://user-images.githubusercontent.com/58088988/183289130-d6bfd4c3-1e12-43f8-be27-59a8429d8311.png)

V2

![image](https://user-images.githubusercontent.com/58088988/183290582-af0a91db-4666-4923-80c9-9a391caca9f7.png)

![image](https://user-images.githubusercontent.com/58088988/183290592-b72dc061-b9e0-439a-ba3e-8e9e0d61ea13.png)

**Comment by AL-Simmons92** [comment]
LFG!!! Wen Tractor!!! FarmerBill In the house!!!

**Reply by eodestroyer** [reply]
YESS!! =)

**Reply by eodestroyer** [reply]
Maybe add a cost to keep the goblin going?  He works for cake?  That way it isn't just a free service but it impacts the constant make and burn.

**Reply by AL-Simmons92** [reply]
Cake!!! Yes

**Comment by Clarkro** [comment]
Love this idea!

**Comment by adamhannigan** [comment, marked as answer]
Wen Tractor!

Love it and on the roadmap - so much to build so little time!

**Reply by Q4FOIL** [reply]
> Wen Tractor!
> 
> Love it and on the roadmap - so much to build so little time!

@adamhannigan if I can help you in other things, just ask

#WenTractor #SunflowerLand

**Comment by Life-Is-Great** [comment]
My main reason for loving this is that it removes bots advantages.
Instead of fighting bots, we can introduce items that do what a bot do directly in the game 😃
